packages/component-library/src/CivicCardStack/CivicCardStack.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'emotion';
+import CivicStoryCard from '../CivicStoryCard/CivicStoryCard';
+
+const lastCardOpacity = 0.4;
+
+const foregroundCard = css`
+`;
+
+const bgCard = (index, total, baseOpacity) => css`
+  position: absolute;
+  z-index: ${-index};
+  left: ${5 * (index)}px;
+  top: ${-5 * (index)}px;
+  opacity: ${1 - (index)*(1- baseOpacity)/(total - 1)};
+`;
+
+const BackgroundStoryCard = (index, total, children) => (
```

Comment:
This is the component that I think could be improved @nrbernard @DingoEatingFuzz